scenes/game/game.gd

```diff
@@ -70,9 +70,11 @@ var select_card_must_be_max_or_min = false
 var select_card_require_force = 0
 var select_card_up_to_force = 0
 var select_card_destination = ""
+var select_boost_can_cancel = false
 var select_boost_valid_zones = []
 var select_boost_limitation = ""
 var select_boost_ignore_costs = false
+var select_boost_from_shortcut = false
```

Comment:
looks like you took the dictionary out for just more variables.
I was suggesting the dictionary since these things all go together and should be set all at the same time and each should also be set together.
However, I don't have strong opinions on this.